### DISCUSSION TITLE
Request for Typst Example

### DISCUSSION THREAD

**Discussion by frettig-radiox** [jgm/pandoc#9832] [Ideas]
Hi everyone,

I’m looking for an example that demonstrates how to import and export to Typst

Frank

**Comment by iandol** [comment]
I write using the excellent writing app Scrivener, which compiles to Pandoc markdown and runs Pandoc. Here is the sample markdown output from Scrivener, the intermediate `.typ` and `pdf` result:

https://raw.githubusercontent.com/iandol/scrivomatic/master/sample-output/Typst.md

https://github.com/iandol/scrivomatic/blob/master/sample-output/Typst.typ

https://github.com/iandol/scrivomatic/blob/master/sample-output/Typst.pdf

https://forum.literatureandlatte.com/t/typst-a-new-markup-page-layout-engine-to-take-on-latex/134674

Note I use some filters to tweak the workflow. For example Typst uses `<fig-xxx>` and `@fig-xxx` for cross-references but this clashes with citeproc and raw html. I use a filter so Pandoc can handle these.

This is all markdown > typst, I don't work the other way round.

**Reply by frettig-radiox** [reply]
Thank you. That helped me to get started.
